sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1356,6 +1357,21 @@ def test_asec():
     assert asec(x).rewrite(acsc) == -acsc(x) + pi/2
 
 
+def test_asec_is_real():
+    assert asec(S.Zero).is_real is None
+    assert asec(1).is_real is True
+    assert asec(-1).is_real is True
+    assert asec(-0.5).is_real is False
+    assert asec(0.5).is_real is False
```

Comment:
Yes, then can be removed. I guess the first one should be left, afterall it was the main issue.